addons/l10n_es_edi_sii/tests/test_resequence.py

```diff
@@ -0,0 +1,136 @@
+from datetime import date
+
+from freezegun import freeze_time
+
+from odoo.exceptions import UserError
+from odoo.tests import Form, tagged
+
+from .common import TestEsEdiCommon, mocked_l10n_es_edi_call_web_service_sign
+
+
+@tagged("post_install_l10n", "post_install", "-at_install")
+class TestResequenceSII(TestEsEdiCommon):
+    @classmethod
+    def setUpClass(
+        cls,
+        chart_template_ref="l10n_es.account_chart_template_full",
+        edi_format_ref="l10n_es_edi_sii.edi_es_sii",
+    ):
+        cls.startClassPatcher(freeze_time("2019-06-01", tick=True))
+        super().setUpClass(
+            chart_template_ref=chart_template_ref, edi_format_ref=edi_format_ref
+        )
+        cls.classPatch(
+            type(cls.env["account.edi.format"]),
```

Comment:
```suggestion
            cls.env.registry['account.move'],
```
is what I see elsewhere